### DISCUSSION TITLE
Tocker's Trials

### DISCUSSION THREAD

**Discussion by PossessedDevil** [TeamFightTacticsBots/Alune#89] [Ideas]
I've noticed that Tocker's Trials gives time played for the pass (tested on Arcane pass 29 November 2024), would it be possible to make it play that so there's 0 risk of other players reporting you? Would be most excelent for farming.
I guess you'd have to make it pick up the money orb and click next round but I'm no programmer so that's where my idea ends xd

**Comment by akshualy** [comment]
Thanks for the discovery and the idea, I'll look into this :)
EDIT
After some testing, it seems the mode does not count twoards play time. I assume it looked like it works for you because play time experience reward for TFT games gets rewarded only after the **entire** TFT lobby you were in finishes.

**Comment by PossessedDevil** [comment]
Ah it only gives first win, you're right it doesn't give pass progress. Sorry o/
